### ISSUE TITLE
Not working with vue 1.0.18

### ISSUE THREAD

**Issue by GuillaumeLeclerc** [GuillaumeLeclerc/vue-google-maps#36]
The component is not compatible with `vue 1.0.18`.

I'm fixing it right now. In the meantime please use `vue 1.0.17`

Thanks for your understanding

**Comment by simplesmiler** [comment]
@GuillaumeLeclerc from `src/components/marker.vue`:
``` js
data() {
  this.mapAvailableDefered = new Q.defer();
  this.mapAvailable = this.mapAvailableDefered.promise;
  this.destroyed = false;
},
```
`data` function should return an object. Probably better to replace this with a `created` hook.

**Comment by GuillaumeLeclerc** [comment]
You are perfectly right ! But I think there are another problems. I will try to fix everything the next week. Thank you for finding this one :)

**Comment by GuillaumeLeclerc** [comment]
Fixed in v0.1.13
